app/admin/views.py

```diff
@@ -83,6 +83,50 @@ def approved_users():
         'admin/approved_users.html', users=users, roles=roles)
 
 
+@admin.route('/view-tags', methods=['GET'])
+@login_required
+@admin_required
+def view_tags():
+    """View and manage all tags."""
+    Tags = Tag.query.all()
```

Comment:
can u sort the tag list by the name of the tag type? so then it doesn't matter that filtering by tag type doesnt work

Reply:
The tags show up together with their respective tag types already. 
![image](https://user-images.githubusercontent.com/16725465/38634043-6fd6112e-3d8f-11e8-9c75-4163c3097af7.png)

Reply:
when you add a new tag, it automatically shows up at the bottom of the list -- it isn't grouped w the others. They're by default ordered by index, and `generate_fake` makes all the tags of the same type sequentially.